sentences/ro/_common.yaml

```diff
@@ -47,18 +47,18 @@ lists:
       - in: "f"
         out: "fahrenheit"
 expansion_rules:
-  area: "{area} [(zona | zonă | regiune)]"
+  area: "{area} [(zona | zonă | regiune | cameră | camera)]"
   brightness: "{brightness} [(la suta | la sută | %)]"
   temperature: "{temperature} [[de] grade] [{temperature_unit}]"
   name: "({name}[ul] | {name})"
   este: "(e | este)"
   modifica: "(schimba | schimbă | modifica | modifică | seteaza | setează)"
   in: "(in | în)"
-  din: "(din | in | în | pentru)"
+  din: "(din | in | în | pentru | de la | de pe )"
   deschide: "(start | pornește | porneste | deschide | aprinde | activează | activeaza)"
   inchide: "(stop | oprește | opreste | închide | inchide | stinge | dezactivează | dezactiveaza)"
   seteaza: "(seteaza | setează | pune | adjusteaza | adjustează | schimba | schimbă | modifica | modifică)"
-  lumina: "(luminile | lumina | becul | becurile)"
+  lumina: "(luminile | lumina | becul | becurile | veioza | veiozele | spotul | spoturile | aplica | aplicele | banda led)"
```

Comment:
Aceasta sintagma este folosita pentru *domeniul* lumina. Intr-adevar, spui "aprinde lumina" sau "inchide becul" si daca e vorba de un LED, o veioza, un spot etc. Insa invers nu prea se intampla, decat daca e numele specific al unui device, caz in care intent-ul contine un `{name}`.

Deci n-as adauga aceste cuvinte

Reply:
Eu am în mai multe camere lumina de sus de la aplică sau lustră și veioză. Dacă spun aprinde lumina, mi se aprinde lumina de sus, pentru veioză trebuie să îi spun explicit ”aprinde veioza”. La fel la bucătărie am lumina de sus din tavan dar și spoturi sau bandă led. Da ai dreptate, trebuie să mă hotărăsc ce să aprindă când îi spun ”aprinde lumina” iar dacă mai am ceva să îi fac solicitare personalizată. În Rhasspy aveam posibilitatea să definesc fiecare entitate cum să îi spun în română și aveam pentru becul de sus ”lumina din bucătărie” și pentru spoturi ”spoturile din bucătărie”. Sper să se poată face asta și în HA. Parcă tu aveai un comentariu pe discord cu o problemă asemănătoare parcă cu ”fereastra din dreapta”.

Reply:
[La asta te referi](https://github.com/home-assistant/intents/issues/574), probabil.

Ideea e asa: cand spui "aprinde lumina in bucatarie", ar trebui ca intent-ul sa fie de aprins toate luminile, dar doar cele din bucatarie. E generic si se refera la o zona.

Cand spui "aprinde veioza din colt din dormitor", te referi la o anumita entitate din domeniul `light`, foarte specifica. Ar trebui ca "veioza" sa fie parte din numele entitatii, nu din sintagma generica.

Motivul pentru care spun asta este ca la un moment dat o sa spui "aprinde veioza" cand in acea camera ai o plafoniera, o banda LED si o veioza, dar se vor aprinde toate pentru ca "veioza" se refera la toate luminile din acea camera, conform propunerii tale de modificare de mai sus. Daca spui "aprinde lumina" sau "aprinde becurile", mi se pare OK sa se aprinda toate. Formularea e suficient de generica cat sa admita asa ceva.

Reply:
Da, ai dreptate,  eu mă gândeam cum vom proceda când avem mai multe lumini într-o cameră și vrem să aprindă una anume. Am văzut ceva comentarii care spuneau că la următorul update se va introduce ”alias” pentru toate entitățile și cred că asta va rezolva problema. Mulțumesc pentru clarificare.